app/src/main/java/com/twilio/video/app/ui/room/RoomActivity.java

```diff
@@ -119,7 +115,7 @@
     private static final int STATS_DELAY = 1000; // milliseconds
     private static final String MICROPHONE_TRACK_NAME = "microphone";
     private static final String CAMERA_TRACK_NAME = "camera";
-    private static final String SCREEN_TRACK_NAME = "screen";
+    static final String SCREEN_TRACK_NAME = "screen";
```

Comment:
Is this referenced somewhere else? Is that why the `private` visibility was removed?

Reply:
These names should only be referenced in the SDK abstraction layer (i.e. RoomManager, LocalParticipantListener, and RemoteParticipantListener). However, since there is still some SDK logic in the RoomActivity that hasn't been abstracted, there are some references in the activity. I'll move these constants to the `RoomManager` for now.